server/views/taskList/taskList.pug

```diff
@@ -31,6 +31,13 @@ block content
             if (['UNSTARTED', 'ELIGIBILITY'].includes(licenceStatus.stage))
                 include eligibilityTask
                 include proposedAddressTask
+                if licenceStatus.decisions.eligible
+                    if licenceStatus.decisions.insufficientTime && !licenceStatus.decisions.insufficientTimeContinue
+                        include caSubmitRefusalTask
```

Comment:
might be better to introduce a new decision `insufficientTimeStop`, based on the combination of insufficientTime and !insufficientTimeContinue, then this would be
` if insufficientTimeStop include caSubmitRefusalTask`